### PR TITLE
Describe contract for webhook payload and response in README

### PR DESCRIPTION
**Pull request by ellieayla**
Describe contract for webhook payload and response in README

Log Insight will retry failing requests, as identified by lack of an HTTP/200 response.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -101,6 +101,15 @@ We’d love to get contributions from you! Once you are up and running,
 take a look at the `Contribution Document`_ to see how to get your
 changes merged in.
 
+Log Insight and vROps make outbound HTTP connections to your shim, with a JSON body containing alerts details.
+The shim is responsible for parsing the body and identifying key information relevant to its operation.
+The shim makes one or more upstream connections to some other service using a native library or raw HTTP, performing a needed task.
+
+At completion, the shim reports back an overall success or failure using the HTTP status code; :code:`200` is treated as success while all other results are treated as a failure.
```
Comment (flands) [inline README.rst]: This is not true today -- the response the shim receives from the third-party is sent back.

Comment (alanjcastonguay) [inline README.rst]: Whether or not HTTP is being used upstream to a third-party, the shim still has a responsibility to assert success or failure and report back accordingly. Most (all?) shims avoid that responsibility today, use HTTP on both sides and pass through the status unchanged. That's a leaky abstraction, and one that should be changed.

Comment (flands) [inline README.rst]: I disagree -- assume you had native webhook support between LI and vRO, is the expectation that vRO return a 200 OK? The shim is a shim not a translator. We may in the short-term before a translation, but this is due to a bug in webhook originator, not a problem in the shim.

Comment (alanjcastonguay) [inline README.rst]: The shim is definitely a translator of the request, and is in a position to translate the response as well. Shims are always specific to the third-party, and are mapping the expectations of two products. For example, the shim could translate the string `success` or the lack of a `UserNotFoundException` into HTTP/200.

That said, the webhook originator accepting all 2xx as Success would definitely be reasonable too. If that's true, this text should change accordingly.

**Comment by flands** [comment]
Where are we with this? It is not true that the shim always returns a 200.

**Pull request closed without merging**